.github/sync-translations.php

```diff
@@ -62,6 +66,24 @@ function dumpXliff1(string $defaultLocale, MessageCatalogue $messages, string $d
     return preg_replace('/^ +/m', '$0$0', $dom->saveXML());
 }
 
+function mergeDom(\DOMDocument $dom, \DOMNode $tree, \DOMNode $input)
+{
+    $new = $dom->createElement($input->tagName);
```

Comment:
Why not using `$new = $dom->adoptNode($input->cloneNode(true))` instead of this `mergeDom` logic that needs to handle each node type separately ?

Reply:
Even simpler: `$new = $dom->importNode($input, true)` which combines the 2 actions

Reply:
That will mess up the indentation. It will double every time you run the script.

Reply:
maybe the logic changing the indentation generated by PHP should run only for the indentation before tags:

```
return preg_replace('/^( +)</m', '$1$0', $dom->saveXML());
```